frontend/src/InteractBar.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { ReactComponent as ListIcon } from "./svg/list.svg";
+import { ReactComponent as AddPinIcon } from "./svg/file-plus.svg";
+import ListView from "./ListView";
+import LocationForm from "./LocationForm";
+
+function ShowListView() {
+  ReactDOM.render(<ListView />, document.getElementById("root"));
+}
+
+function ShowLocationForm() {
+  ReactDOM.render(<LocationForm />, document.getElementById("root"));
+}
+
+
```

Comment:
Maybe run prettier to catch multiple new lines?

Reply:
Understood. Will do.